### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by vlmaksime**
Update russian translation in settings

### CODE REVIEW

File: resources/language/resource.language.ru_ru/strings.po [hunk 1]
```diff
@@ -1082,16 +1090,3 @@ msgstr "IP клиента %s"
...
-msgstr "Воспроизвести с субтитрами"
-
-msgctxt "#30703"
-msgid "Are you sure?"
-msgstr "Вы уверены?"
-
-msgctxt "#30704"
-msgid "Use YouTube website urls"
-msgstr "Использовать URLs сайта YouTube"
-
```
Comment (anxdpanic) [inline resources/language/resource.language.ru_ru/strings.po]: Is there a reason why these 30702-30704 are removed? Also missing 30705

**Comment by anxdpanic** [comment]
Thank you for the contribution, just the one comment

File: resources/language/resource.language.ru_ru/strings.po (hunk 1 above)
Comment (vlmaksime) [reply resources/language/resource.language.ru_ru/strings.po]: There is no such strings at [6.x.x/resources/language/resource.language.en_gb/strings.po](https://github.com/jdf76/plugin.video.youtube/blob/6.x.x/resources/language/resource.language.en_gb/strings.po)
Or I have to use file from master branch?

Comment (anxdpanic) [reply resources/language/resource.language.ru_ru/strings.po]: Yes, sorry I missed that this one was against 6.x.x. Please PR this against master

**Comment by vlmaksime** [comment]
Done

**Pull request merged**